gips/data/gpm/gpm.py

```diff
@@ -148,58 +162,67 @@ def query_provider(cls, asset, tile, date):
         for filename in filenames:
             if compiled_pattern.search(filename):
                 if asset == 'IMERG-DAY-FINAL':
                     return filename, None
                 elif asset == 'IMERG-DAY-LATE':
-                    end_timestamp = int(re.search(r'(?<=E)[0-9]{6}', filename).group(0))
+                    end_timestamp = int(re.search(r'(?<=E)[0-9]{6}', filename)
+                                        .group(0))
                     if end_timestamp > 230000:
                         return filename, None
                     else:
                         continue
                 elif asset == 'IMERG-DAY-EARLY':
-                    end_timestamp = int(re.search(r'(?<=E)[0-9]{6}', filename).group(0))
+                    end_timestamp = int(re.search(r'(?<=E)[0-9]{6}', filename)
+                                        .group(0))
                     if end_timestamp == 65959:
                         return filename, None
                     else:
                         continue
                 elif asset == 'IMERG-DAY-MID':
-                    end_timestamp = int(re.search(r'(?<=E)[0-9]{6}', filename).group(0))
+                    end_timestamp = int(re.search(r'(?<=E)[0-9]{6}', filename)
+                                        .group(0))
                     if end_timestamp == 145959:
                         return filename, None
                     else:
                         continue
                 elif asset == '3B42-DAY-LATE':
-                    end_timestamp = int(re.search(r'[0-9]{2}(?=\..)', filename).group(0))
+                    end_timestamp = int(re.search(r'[0-9]{2}(?=\..)', filename)
+                                        .group(0))
                     if end_timestamp == 21:
                         return filename, None
                     else:
                         continue
         return None, None
 
-
     @classmethod
     def fetch(cls, asset, tile, date):
         utils.verbose_out('%s: fetch tile %s for %s' % (asset, tile, date), 3)
         qs_rv = cls.query_service(asset, tile, date)
         if qs_rv is None:
             return []
         asset_fn = qs_rv['basename']
-        with utils.error_handler("Error downloading from " + cls._assets[asset]['host'], continuable=True):
-            ftp = cls.ftp_connect(asset, date)  # starts chdir'd to the right directory
+        with utils.error_handler("Error downloading from " +
+                                 cls._assets[asset]['host'], continuable=True):
+            ftp = cls.ftp_connect(asset, date)  # starts chdir'd to the right
+            # directory
             stage_dir_fp = cls.Repository.path('stage')
             stage_fp = os.path.join(stage_dir_fp, asset_fn)
-            with utils.make_temp_dir(prefix='fetchtmp', dir=stage_dir_fp) as td_name:
+            with utils.make_temp_dir(prefix='fetchtmp', dir=stage_dir_fp)\
+                    as td_name:
```

Comment:
You don't have to change this line, but when I'm linebreaking I like to avoid backslashes.  For instance:

```
with utils.make_temp_dir(
        prefix='fetchtmp', dir=stage_dir_fp) as td_name:
    temp_fp = os.path.join(td_name, asset_fn)
    # etc etc
```